From 8dd8b0bd4680f34e715ff3a6668713c46396c301 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Wed, 2 Oct 2019 14:56:36 -0300
Subject: [PATCH 2/2] Remove unecessary BBL console output

Remove "bbl loader" duriing initialization.
Remove "Power off" during shutdown.
---
 machine/minit.c | 2 +-
 machine/mtrap.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/machine/minit.c b/machine/minit.c
index 5ad6d92..0b7d5e6 100644
--- a/machine/minit.c
+++ b/machine/minit.c
@@ -159,7 +159,7 @@ void init_first_hart(uintptr_t hartid, uintptr_t dtb)
   query_uart(dtb);
   query_uart16550(dtb);
   query_htif(dtb);
-  printm("bbl loader\r\n");
+  /* printm("bbl loader\r\n"); */
 
   hart_init();
   hls_init(0); // this might get called again from parse_config_string
diff --git a/machine/mtrap.c b/machine/mtrap.c
index 42006b8..99f98db 100644
--- a/machine/mtrap.c
+++ b/machine/mtrap.c
@@ -239,7 +239,7 @@ void trap_from_machine_mode(uintptr_t* regs, uintptr_t dummy, uintptr_t mepc)
 
 void poweroff(uint16_t code)
 {
-  printm("Power off\r\n");
+  /* printm("Power off\r\n"); */
   finisher_exit(code);
   if (htif) {
     htif_poweroff();
-- 
2.17.1

